packages/cover-traffic-daemon/src/index.ts

```diff
@@ -32,20 +50,21 @@ const argv = yargs(process.argv.slice(2))
   .wrap(Math.min(120, terminalWidth()))
   .parseSync()
 
-async function generateNodeOptions(): Promise<HoprOptions> {
+async function generateNodeOptions(environment: ResolvedEnvironment): Promise<HoprOptions> {
   const options: HoprOptions = {
-    provider: argv.provider,
+    announce: false,
     createDbIfNotExist: true,
-    password: '',
+    environment,
     forceCreateDB: true,
-    announce: false
+    password: ''
```

Comment:
Empty passwords are known to cause problems
```suggestion
    password: 'ct-node'
```

Reply:
should be taken care of by the CT working group

Reply:
Plaintext pwd. Love it 😝